src/generic/triangle_mesh.h

```diff
@@ -90,25 +90,42 @@ namespace oomph
       std::ofstream outfile;
       setup_boundary_element_info(outfile);
     }
-
+   
+   /// Timestepper used to build elements
+   TimeStepper* timestepper_pt()
+    {
+     return Time_stepper_pt;
+    }
+   
+   
     // [zdec] work in progress
     /// Divide an element into three by creating edges from the centroid to
     /// each of the vertex nodes. Take the old element pointer and fill out
     /// vectors containing pointers to the new three elements and all the
-    /// nodes. Optionally takes a pointer to a timestepper to pass on to the
-    /// new elements and nodes.
+    /// nodes.
     template<class ELEMENT>
     void split_element_through_centroid(
       FiniteElement* const& el_pt,
       Vector<FiniteElement*>& new_el_pt,
-      Vector<Node*>& new_nod_pt,
-      TimeStepper* const& time_stepper_pt = &Mesh::Default_TimeStepper);
+      Vector<Node*>& new_nod_pt);
 
     /// Make sure no elements have two boundary edges by splitting them through
     /// their centroid (e.g. no corner elements straddling two boundaries).
     template<class ELEMENT>
-    void split_elements_with_multiple_boundary_edges(
-      TimeStepper* const& time_stepper_pt = &Mesh::Default_TimeStepper);
+    void split_elements_with_multiple_boundary_edges()
```

Comment:
Have we lost the ability to pass a timestepper to the new elements? It might be a niche use case, but the timestepper we wish the elements to use might not be the mesh default.

Reply:
Good question: how do newly created nodes get their timestepper? May need to add the timestepper again.